src/lib/env.js

```diff
@@ -40,21 +38,12 @@ export class Env {
     return parser.parse.call(parser, emit, data);
   }
 
-  _createEntries(lang, ast) {
-    const entries = Object.create(null);
-    const create = lang.src === 'qps' ?
-      createPseudoEntry : createEntry;
-
-    for (let i = 0, node; node = ast[i]; i++) {
-      const id = node.$i;
-      if (id in entries) {
-        this.emit('duplicateerror', new L10nError(
-         'Duplicate string "' + id + '" found in ' + lang.code, id, lang));
```

Comment:
hmmm... the last argument is reasonable. I'm still scared of accidental overrides so let's make sure that we remember that resource order in a view is significant.
We should also probably fix BTO to not report them then as well.